color/p5js/Color-Continuity/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Law of Prägnanz, Law of Contrast, Law of Multistable-perception, Law of Preference
+tags: [P5.js, Law of Prägnanz, Law of Law of Similarity, Law of Common-Fate, Law of Connected-Elements, Law of Contrast, Law of Multistable-perception, Law of Relativity of Properties, Law of Similarity, Law of Simplicity, Law of Symmetry, Color, Chance, Interaction, Repeat]
```

Comment:
- „Common Fate“ and „Connected Elements“  without hypens.
- categories in lowercase.

Don't you think these are to many laws?